logrotate/load-and-sleep.sh

```diff
@@ -0,0 +1,14 @@
+
+#!/bin/sh
```

Comment:
This should be the first line. Remove the blank line on the top.

Reply:
Working fine after removing blank line on top